TSPractice/ts-calculator/src/script.ts

```diff
@@ -127,6 +133,6 @@ document.onkeydown = function (event: KeyboardEvent) {
         }
       }
     }else{
-        alert("Invalid Key Press!!");
+        // do nothing
```

Comment:
If this else is not required, shall we remove it

Reply:
yes ma'am, I will do that, I put there, because of alert box, no problem